x-pack/plugins/endpoint/server/routes/alerts.ts

```diff
@@ -5,27 +5,107 @@
  */
 
 import { IRouter } from 'kibana/server';
+import { RequestHandler } from 'kibana/server';
+import { SearchResponse } from 'elasticsearch';
+import { schema } from '@kbn/config-schema';
 
-import json from './sampledata.json';
+import { kibanaRequestToAlertListQuery } from '../services/endpoint/alert_query_builders';
+import { AlertData, AlertResultList } from '../../common/types';
+import { EndpointAppContext } from '../types';
+
+const ALERTS_ROUTE = '/api/endpoint/alerts';
+
+export function registerAlertRoutes(router: IRouter, endpointAppContext: EndpointAppContext) {
+  const alertsHandler: RequestHandler<any, any, any> = async (ctx, req, res) => {
```

Comment:
Might be worth trying to avoid `any` since it effectively disables type safety. If you decide to try, let me know if ya want help w/ it

Reply:
Updated!